repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -485,16 +486,20 @@ def map_repositories(packages):
         del packages[pkg]
 
     if repo_without_mapping:
-        report_skipped_packages('packages will not be installed or upgraded due to repositories unknown to leapp:',
-                                repo_without_mapping,
-                                "Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of "
-                                "the Red Hat Enterprise Linux 7 product.")
+        report_skipped_packages(
+            title='Packages from unknown repositories may be not installed',
+            message='packages may be not installed or upgraded due to repositories unknown to leapp:',
```

Comment:
```suggestion
            message='packages may be not be installed or upgraded due to repositories unknown to leapp:',
```

Reply:
I'm sorry, the suggestion was incorrect. I did not read it properly. I though there's "be" missing but it was actually just misplaced.